recipes/ddrage/meta.yaml

```diff
@@ -0,0 +1,81 @@
+package:
+  name: ddrage
+  version: "1.1.4"
+
+source:
+  fn: ddrage-1.1.4.tar.gz
+  url: https://pypi.python.org/packages/46/e2/fb9f89336197106198c0ac659e39b0530eaee13f1ac31ab885262f35f0e2/ddrage-1.1.4.tar.gz
+  md5: 73e90e01ccbbb0f046a5dc664f8a50d6
+#  patches:
+   # List any patch files here
+   # - fix.patch
+
+build:
+  # noarch_python: True
+  # preserve_egg_dir: True
+  skip: True # [py27]
+  entry_points:
+    # Put any entry points (scripts to be generated automatically) here. The
+    # syntax is module:function.  For example
+    #
+    # - ddrage = ddrage:main
+    #
+    # Would create an entry point called ddrage that calls ddrage.main()
+
+    - rage = ddrage.__main__:main
+    - randomize_fastq = ddrage.tools.randomize_fastq:main
+    - learn_qmodel = ddrage.tools.learn_qmodel:main
+
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  # number: 1
+
+requirements:
+  build:
+    - python >=3.5
```

Comment:
Can you remove the pinning here. You already skip python2 builds in your build section.